### ISSUE TITLE
Update service request form link across ReportStream website

### ISSUE THREAD

**Issue by jillian-hammer** [CDCgov/prime-reportstream#16135]
## Problem statement

The Support team needed to update their Smartsheet service request form, which resulted in a new URL being generated. This ticket captures the work of changing the URL to the updated service request form across the site.

## What you need to know

This is a follow on ticket to design's audit of the site #15980. 
NEW URL: `https://app.smartsheetgov.com/b/form/95d2f16b772c42d8aaa0ef5db28a6838`

## Acceptance criteria
Checklist of all the link locations that need to be replaced:

Your connection (`/managing-your-connection`)

- [ ] For healthcare orgs: "... [contact us] to create an account"
- [ ] For public health agencies: "... [contact us] to create an account"
- [ ] The submission history API: "[Contact us] to learn more"
- [ ] Ongoing support: "[Submit a service request]"

Support landing page  (`/support`)
- [ ] Support request: "For issues with an existing connection, [submit a ticket.]"
- [ ] Getting started > How do I register for a ReportStream account? > "Submit a [service request] with information about your organization..."
- [ ] Using ReportStream > Can public health departments receive a report on facilities reporting them? > "[Contact us] if you need an account or help viewing the dashboard"
- [ ] Using ReportStream > How do I stop receiving negative test results? > "[Contact us] so we can update..."
- [ ] Using ReportStream > How do I resubmit my data? > "[Submit a support request,] and our team will..."
- [ ] Using ReportStream > How do I manage team member accounts? > "To add or remove team members, [contact us]."

## To do

- [ ] Update links using the above checklist
- [ ] Notify Design (cc @jillian-hammer) and the Support team (cc @TaneaY) when the links are all updated

**Comment by jillian-hammer** [comment]
hi @MikaelahD13 and @chris-kuryak - here is the eng follow-up ticket for updating the service request link. can we prioritize for this sprint 105? thanks!

**Comment by chris-kuryak** [comment]
Thanks! Added to the roadmap for next sprint.

**Comment by chris-kuryak** [comment]
@etanb Did this change get deployed on Thursday? If not, should it get deployed by Tues?

Just checking so we can update Support Team to let them know when they can deprecate the old form link.
cc: @MikaelahD13 @jillian-hammer

**Comment by etanb** [comment]
@chris-kuryak no, you can see the PR just got approved today, so will just have gotten merged today. Will go out next week.

**Comment by etanb** [comment]
@chris-kuryak no, you can see the PR just got approved today, so will just have gotten merged today. Will go out next week.

**Comment by chris-kuryak** [comment]
Thank you for confirming, Etan.

@TaneaY expect the links to the new support ticket form to be live next Tues 10/15, assuming deployment is successful. 

cc: @jillian-hammer @MikaelahD13
